birthday.h

```diff
@@ -0,0 +1,53 @@
+#include "config.h"
+/**
+ * Define a struct objec to hold birthday data
+ */
+struct Birthday{
+  Birthday(){};
```

Comment:
No need for trivial constructors, you can create structs with this syntax
```
Birthday birthday = {
    .month=1,
    .day=2,
    .year=2000,
    .name="name",
};
```

Reply:
Are you referring to only the default constructor or the entire struct definition?  The struct isn't in general trivial as it is reused for each birthday for reusability and repeated methods within it.

Reply:
Are you referring to only the default constructor or the entire struct definition? The struct isn't in general trivial as it is reused for each birthday for reusability and repeated methods within it.

Reply:
Just the constructors, not the definition of the struct itself. That code I provided is a c++ syntax for struct instantiation. Both ways work but unless a struct's constructor is actually doing things other than just setting properties I prefer that syntax.